wifiphisher/pywifiphisher.py

```diff
@@ -134,7 +139,14 @@ def parse_args():
               "to believe it is within an area that was previously captured "
               "with --lure10-capture. Part of the Lure10 attack."))
     parser.add_argument(
-        "--logging", help=("Log activity to file"), action="store_true")
+        "--logging",
+        help="Log activity to file",
+        action="store_true")
+    parser.add_argument(
+        "-lP",
+        "--logpath",
+        default=None,
+        help="Full path and filename of the logfile")
```

Comment:
A "path" typically includes the "filename". So I would change this to "Determine the full path of the logfile."